lib/Driver/Job.cpp

```diff
@@ -403,6 +407,30 @@ void Job::printSummary(raw_ostream &os) const {
   os << "}";
 }
 
+static void
+writeResponseFile(llvm::raw_ostream &OS, const llvm::opt::ArgStringList &Arguments) {
+  // wrap arguments in quotes to ensure compatibility with Unix and Windows
```

Comment:
Nitpick: Please use complete sentences in comments, including capitalization and a period at the end.